python/ray/serve/router.py

```diff
@@ -87,6 +88,12 @@ def __init__(
             {"deployment": self.deployment_name}
         )
 
+    def _reset_replica_iterator(self):
```

Comment:
add docstring with the behavior here (what happens to inflight & subsequent requests)